src/components/Settings.vue

```diff
@@ -94,6 +98,7 @@ export default {
     maxWidth: 100,
     minHeight: 1,
     maxHeight: 100,
+    colors: [],
```

Comment:
Don't forget to have the 17 common ones selected by default. Here's a quick way to do that:

```suggestion
    colors: Array(17).fill().map((_, i) => i),
```